vmdb/app/models/host.rb

```diff
@@ -1400,6 +1399,14 @@ def connect_ssh(options={})
       rl_user, rl_password = self.auth_user_pwd(:root)
       su_user, su_password = nil, nil
     end
+    return rl_user, rl_password, su_user, su_password, {}
+  end
+
+  def connect_ssh(options={})
+    require 'MiqSshUtil'
+
+    rl_user, rl_password, su_user, su_password, additional_options = ssh_users_and_passwords
```

Comment:
Yes, see spec/models/host_spec.rb for the Host tests and spec/models/host_vmware_spec.rb for a Host specific example (vmware).  My thought would be to add the test for the base host method in the host_spec.rb and we create a new host_openstack_infra_spec.rb to test the overloaded method.